js/src/ui/Tooltips/Tooltip/tooltip.js

```diff
@@ -30,15 +31,15 @@ let tooltipId = 0;
 
 class Tooltip extends Component {
   static propTypes = {
-    title: PropTypes.string,
-    text: PropTypes.string,
-    right: PropTypes.bool,
+    className: PropTypes.string,
     currentId: PropTypes.number,
     maxId: PropTypes.number,
-    className: PropTypes.string,
+    right: PropTypes.bool,
```

Comment:
We should, the only reason I have not pulled it in as part of one of my eslint updates (yet) is that we are basically wrong in 70% of the cases. So lot of small little niggly updates, just didn't have the time to fiddle.

Medium-term, I would love to.